source/user-manual/capabilities/vulnerability-detection/offline-update.rst

```diff
@@ -361,50 +361,24 @@ Alternatively, they also can be loaded from a local path as follows:
 National Vulnerability Database
 -------------------------------
 
-To perform an offline update of the National Vulnerability Database, you must request its feed stating a starting year. Wazuh provides an `update script <https://github.com/wazuh/wazuh/blob/4.4/tools/vulnerability-detector/nvd-generator.sh>`__ that automates the process of downloading the feed and checking for server downtime.
-
-How to use the update script
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-#. Create a directory for the feeds.
-
-   .. code-block:: console
-
-      # mkdir /local_path/nvd-feed
-
-#. Run the script setting the starting year, of which the earliest is 2002, and the target path.
-
-   .. code-block:: console
-
-      # nvd-generator.sh 2002 /local_path/nvd-feed
-
-   The script may output an error message like the following.
-
-   .. code-block:: console
-      :class: output
-
-      Page download failed (504), retrying...
-
-   The error message indicates that the National Vulnerability Database servers may be temporarily unavailable to you. The script keeps trying to finish the download until it acquires the full feed.
-
-Finally, the feed is in numbered files whose names follow the format ``nvd-feed<number>.json.gz``. You must extract the compressed files to be able to use them. To update locally, you must set the path of those files with a regular expression as shown below.
+To perform an offline update of the National Vulnerability Database, you must get the feed from `complete_nvd_feed.json.gz <https://feed.wazuh.com/vulnerability-detector/NVD/generated-feeds/complete_nvd_feed.json.gz>`__ (`sha256 <https://feed.wazuh.com/vulnerability-detector/NVD/generated-feeds/complete_nvd_feed.meta>`__) and place it in a user-defined repository.
 
 .. code-block:: xml
 
    <provider name="nvd">
-       <enabled>yes</enabled>
-       <path>/local_path/nvd-feed/nvd-feed[[:digit:]]\{4\}\.json\.gz$</path>
-       <update_interval>1h</update_interval>
+      <enabled>yes</enabled>
+      <url>http://local_repo/complete_nvd_feed.json.gz</url>
+      <update_interval>1h</update_interval>
    </provider>
 
-If you want to upload these files to a local server, they must follow the same numeric sequence that the script downloaded them in. To indicate the numerical range, you must indicate their position with the ``[-]`` tag helped by the ``start`` and ``end`` attributes. For example, you can use the configuration below if the files are from 2015 to 2020.
+To update locally, you have to set the path of the downloaded files with a POSIX regular expression as such:
```

Comment:
```suggestion
To update locally, set the path of the downloaded files using a POSIX regular expression, for example:
```

Reply:
Should I also change the MSU section? I copy paste the description from there:

https://github.com/wazuh/wazuh-documentation/blob/74d70888e9305f077478b1037a4929f3905a43ad/source/user-manual/capabilities/vulnerability-detection/offline-update.rst?plain=1#L399

Reply:
It's not necessary to merge this PR, but it's a good opportunity to improve the text if you want to.